phaser/game.js

```diff
@@ -260,6 +260,7 @@ function startSocketActions(self, allowedPlayersCount) {
     self.add.text(225, 400, 'Game In Progress. Go Away.'.toUpperCase(), { fontSize: '32px' })
   })
   self.socket.on('currentPlayers', function (players) {
+    debugger
```

Comment:
I assume this snuck in here? Do we need the console log below as well?

Reply:
I removed the debuggers and the extra console.log